tests/cases/fourslash/importNameCodeFixDefaultExport2.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="fourslash.ts" />
+
+// @allowJs: true
```

Comment:
Is it necessary that this test be JS? Should work exactly the same in TS?

Reply:
This test matches the repro steps in the bug.  If a TS test would be preferable, I can change it (it should be equivalent).

Reply:
If it's exactly the same with these lines removed I would do that.

Reply:
Assuming you also meant to update the file extensions (so that diagnostics are produced at all), it would still not be exactly the same since the suppression fixes would not be offered (though that distinction is not important to the test/change).

Reply:
Yeah, I agree it's OK to remove the JS-specific suppression codefixes from this test since that's not what the point of the test is.